src/sentry/grouping/strategies/message.py

```diff
@@ -151,36 +154,50 @@
 
 _parameterization_regex = re.compile(_parameterization_regex_str)
 
+# UniqID logic
+encoding = tiktoken.get_encoding("cl100k_base")
+
+
+def num_tokens_from_string(token_str: str) -> int:
+    """Returns the number of tokens in a text string."""
+    num_tokens = len(encoding.encode(token_str))
+    return num_tokens
+
+
+def is_probably_uniq_id(token_str: str):
```

Comment:
Please add return types to your functions. It helps with proper type-checking, otherwise, it assumes it's `Any`.